test/web/serverSetup.js

```diff
@@ -34,65 +34,79 @@ module.exports = async function(){
         if (request.url().includes(`https://${mocks.server}/apiv1`)) {
             // Post requests are normal xhr/call methods
             let payload = '';
-            if(request.method() === 'POST'){
+            if (request.method() === 'POST') {
                 let body = parseRequest(request.postData());
                 switch (body.method) {
                     case 'Authenticate':
                         // Alternate the credential response to test forget()
-                        if(authenticationAttempts%2===0){
-                            payload = { result: mocks.credentials};
+                        if (authenticationAttempts % 2 === 0) {
+                            payload = { result: mocks.credentials };
                             authenticationAttempts++;
                         } else {
-                            payload = { result: mocks.refreshedCredentials};
+                            payload = { result: mocks.refreshedCredentials };
                             authenticationAttempts++;
                         }
                         break;
                     case 'Get':
                         switch (body.params.typeName) {
                             case 'Device':
-                                payload = { result: [mocks.device]};
+                                payload = { result: [mocks.device] };
                                 break;
                             case 'User':
-                                payload = { result: [mocks.user]};
+                                payload = { result: [mocks.user] };
                                 break;
                         }
                         break;
                     case 'Geet':
                         // Poorly formed request tests
-                        payload = { result: {
-                            name: "InvalidCall",
-                            message: "Bad info entered"
-                        }}
+                        payload = {
+                            error: {
+                                code: -32000,
+                                name: "JSONRPCError",
+                                message: "The method 'Geet' could not be found. Verify the method name and ensure all method parameters are included.",
```

Comment:
"Get"

Reply:
Hard to tell from the diff (and maybe by the method name choosen) but this is intentional. The method called is 'Geet'.